src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py

```diff
@@ -1047,6 +1043,30 @@ def handle_corpus_pruning_failures(output: uworker_msg_pb2.Output):
   data_handler.update_task_status(task_name, data_types.TaskState.ERROR)
 
 
+def _fill_backup_urls(fuzz_target: data_types.FuzzTarget,
```

Comment:
Can we pick a better name than "fills" maybe "create"?

Reply:
Done